src/animatedRe.rei

```diff
@@ -740,4 +739,4 @@ let stop: Animation.t => unit;
 let start: (Animation.t, ~callback: Animation.endCallback=?, unit) => unit;
 
 /** See {!Animation.reset} for details. */
-let reset: Animation.t => unit;
\ No newline at end of file
+let reset: Animation.t => unit;
```

Comment:
This change is because I used GitHub UI for the fix :/

Reply:
I think this is correct anyway